app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -121,6 +122,8 @@ fun Fragment.safeNavigateToNewMessageActivity(draftMode: DraftMode, messageUid:
     )
 }
 
+fun List<Message>.getLastMessageToExecuteAction(): Message = lastOrNull { !it.isDraft } ?: last()
```

Comment:
If there are only drafts in the list, will it return a draft? Is this a desired behavior?
If not, we might as well put `last { !it.isDraft }` directly and we'll have a crash if it happens.

Reply:
If there's only Drafts, we want to get the last one. This is the desired behavior.

Example: Any action on a Thread in the Draft folder. They will all be actions on Thread of a single Message, with `isDraft = true`.